pkgs/development/libraries/libbfd/default.nix

```diff
@@ -11,17 +11,20 @@ stdenv.mkDerivation rec {
 
   patches = binutils-raw.bintools.patches ++ [
     ../../tools/misc/binutils/build-components-separately.patch
+    (fetchpatch {
+      url = "https://raw.githubusercontent.com/mxe/mxe/e1d4c144ee1994f70f86cf7fd8168fe69bd629c6/src/bfd-1-disable-subdir-doc.patch";
+      sha256 = "0pzb3i74d1r7lhjan376h59a7kirw15j7swwm8pz3zy9lkdqkj6q";
+    })
   ];
 
   # We just want to build libbfd
-  postPatch = ''
+  postAutoreconf = ''
     cd bfd
   '';
 
   nativeBuildInputs = [ autoreconfHook264 bison ];
   buildInputs = [ libiberty zlib ];
 
-  configurePlatforms = [ "build" "host" ];
```

Comment:
Err its still good to have these in all cases, I think. I'd do that for all autoconf-based packages if I could easily tell them apart.

Reply:
Oh? Hmm sorry.  I thought configurePlatforms was .. .oh, it's not set on cross is it? Okay I'll restore this.

Reply:
It gets set to that on cross, but I rather have it always set that way, for packages that support it. See the `mkDerivation` definition for this.